### PR TITLE
Document how to get started with Campaigns and add note about webhooks

### PR DESCRIPTION
**Pull request by mrnugget**
See https://github.com/sourcegraph/customer/issues/13#issuecomment-593822292 for context on this.

I wanted to add only a note about webhooks but found out that we don't really have documentation for Bitbucket Server webhooks. I'll try to follow up on that.

### CODE REVIEW

File: doc/user/campaigns.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
 # Campaigns
 
-> The campaigns feature was formerly known as "automation".
+> Campaigns are currently available in private beta for select enterprise customers. (This feature was previously known as "Automation".)
```
Comment (eseliger) [inline doc/user/campaigns.md]: maybe make it bold as well? I skimmed over it in the first place and looked for where it mentions that it's not GA 🙈

File: doc/user/campaigns.md [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
-[Sourcegraph code change management](https://about.sourcegraph.com/product/code-change-management) lets you make large-scale code changes across many repositories and different code hosts.
+Campaigns are part of [Sourcegraph code change management](https://about.sourcegraph.com/product/code-change-management) and let you make large-scale code changes across many repositories and different code hosts.
 
-**Important**: If you're on Sourcegraph 3.12 or older, you might also want to look at the old documentation: "[Campaigns documentation for Sourcegraph 3.12](https://docs.sourcegraph.com/@3.12/user/automation)"
+You provide the code to make the change and Campaigns provide the plumbing to turn it into a large-scale code change campaign and monitor its progress.
+
+## Getting started with Campaigns
+
+1. Make sure that the Campaigns feature flag is enabled: [Configuration](#Configuration)
+1. For optimal syncing performance between your code host and Sourcegraph, setup the webhook integration:
```
Comment (eseliger) [inline doc/user/campaigns.md]:
```suggestion
1. Optional: For optimal syncing performance between your code host and Sourcegraph, setup the webhook integration:
```

Comment (eseliger) [reply doc/user/campaigns.md]: it wasn't a 1000% clear to me that the sentence means it is optional to set it up

Comment (mrnugget) [reply doc/user/campaigns.md]: Well, it shouldn't be. I do highly recommend to setup the webhook integration. The condition is exactly what it says: "For optimal syncing performance"

**Pull request merged**